docs/how_to_guides/hosting_validator_models.md

```diff
@@ -0,0 +1,210 @@
+# Hosting Validator Models
+
+Validation using machine learning models is a highly effective way of detecting LLM hallucinations that are not easily or possibly detected using traditional coding techniques. We’ve selected and tuned some ML models to run different validations. These models are wrapped within model validators. This makes the access pattern straightforward — you can use model-based validators the same as any other [validator](https://www.guardrailsai.com/docs/concepts/validators).
```

Comment:
nit links to https://www.guardrailsai.com/docs should be relative and not hard coded to this domain so local and staging can mostly work